plugin/builtin/buildbaron/build_baron.go

```diff
@@ -219,12 +219,12 @@ func (bbp *BuildBaronPlugin) buildFailuresSearch(w http.ResponseWriter, r *http.
 		return
 	}
 
-	fallback := jiraSuggest{bbProj, bbp.jiraHandler}
-	altEndpoint := altEndpointSuggest{bbProj}
+	fallback := &jiraSuggest{bbProj, bbp.jiraHandler}
```

Comment:
From talking with Brian, turns out this is fine, but there is a goroutine leak. (which is ultimately what caused the race to fail). Will look into this and get back to you